starfish/imagestack/parser/tileset/_parser.py

```diff
@@ -25,6 +112,7 @@ def __init__(self, tileset: TileSet) -> None:
                 z=tile.indices.get(Indices.Z, 0))
             self.tiles[key] = tile
         self._extras = tileset.extras
+        self._expectations = _Expectations()
```

Comment:
I'd doc here why you're initializing `_Expectations()` and what the expected behavior is.